textpipe/doc.py

```diff
@@ -375,3 +375,35 @@ def keyterms(self):
         [('awesome', 0.32456160227748454), ('capital', 0.32456160227748454), ('amsterdam', 0.17543839772251532)]
         """
         return self.extract_keyterms()
+
+    @property
+    def vectors(self):
+        """
+        Returns word embeddings for the words in the document.
+        The used spacy models don't have "true" word vectors
+        but only context-sensitive tensors that are within the document.
+
+        Returns:
+        A dictionary mapping words from the document to a 4-tuple with the
```

Comment:
Why do we choose to use a 4 tuple? I guess a dict could be more descriptive.

Reply:
I'd prefer a named tuple over a dict; we could just copy the SpaCy naming (`has_vector`, `vector_norm`, etc) for transparency. This would result in this if you want to retrieve a vector: `doc.vectors['sentence'].vector`

Reply:
> What would that look like in serialized form?

Only option is to serialize it as a dict is guess (through `as_dict()`)

Reply:
Maybe a dict makes a bit more sense then? We have a few more places were we return a tuple and a (list of) dict(s) makes more sense (`ents`, `emojis`, etc.). Should we add that as an issue?

Reply:
A `namedtuple` serializes as list by default: 
```python
>>> from collections import namedtuple
>>> Point = namedtuple('Point', ['x', 'y'])
>>> import json
>>> json.dumps(Point(1, 2))
'[1, 2]'
```
We will represent this as a `dict`, not as a `namedtuple`.